### PR TITLE
Fix bug: Can't change userNameAttributeName for azure sso.

### PR DESCRIPTION
**Pull request by chenrujun**
Fix bug: Can't change userNameAttributeName for azure sso.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 3.4.0-beta.1 (Unreleased)
+- Fix bug that user-name-attribute cannot be configured.(#20209)
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:4]: add link to this issue?

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 3.4.0-beta.1 (Unreleased)
+- Fix bug that user-name-attribute cannot be configured.(#20209)
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter/CHANGELOG.md:4]: same here.

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md (hunk 1 above)
Comment (saragluna) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:4]: Also, should we add this under a `bug fixes` section?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md]: @saragluna , updated.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**